cluster/gce/gci/configure-helper.sh

```diff
@@ -2155,6 +2155,12 @@ EOF
     update-event-exporter ${event_exporter_yaml}
     fluentd_gcp_version="${FLUENTD_GCP_VERSION:-0.2-1.5.28-1}"
     sed -i -e "s@{{ fluentd_gcp_version }}@${fluentd_gcp_version}@g" "${fluentd_gcp_yaml}"
+    if [[ "${LOGGING_STACKDRIVER_RESOURCES:-}" == "new" ]]; then
+      fluentd_k8s_compatibility="true"
+    else
+      fluentd_k8s_compatibility="false"
+    fi
+    sed -i -e "s@{{ fluentd_k8s_compatibility }}@${fluentd_k8s_compatibility}@g" "${fluentd_gcp_yaml}"
```

Comment:
I really don't like doing all this inline, please extract it to separate function, maybe with the previous couple of lines, to make a logical block like `setup-fluentd`

Reply:
PTAL